### DISCUSSION TITLE
License doesn't transfer between computers

### DISCUSSION THREAD

**Discussion by aizotov** [rsayfutdinov/Pikka#44] [General]
Greetings, 
I've previously purchased this software and now have moved to a new machine. Unfortunately, it does not appear that the license has carried over - it once again prompts me to pay for it. How would you recommend resolving this?

**Comment by rsayfutdinov** [comment]
Hey Andrey,
I'm sorry for not getting back to you sooner.

The best way to do it is to download the app from the Mac App Store.
Here is a promo code for you: RWWWNN63AXL3
You can redeem it in the App Store, so you don't need to pay for the app again.

Regards,
Ruslan.

**Comment by aizotov** [comment]
Thank you very much 🙏

Best,
Andrey Izotov

**Comment by Paul-Huntsberry** [comment]
I'm having the same issue. How do I transfer my license from my old computer to the new one?

**Comment by RBHannam** [comment]
Hi - I'm having this problem too. I raised it as an issue a few weeks ago, but there has been no response. Please could you just tell me whether you intend to provide a solution? Many Thanks.
